src/build/patches.ts

```diff
@@ -54,10 +55,10 @@ function handleEnum(node: Node, enums: Record<string, Enum>) {
 }
 
 /**
- * Handles a mixin node by extracting its name and associated events.
+ * Handles a mixin node by extracting its name and associated events and properties.
```

Comment:
```suggestion
 * Handles a mixin node by extracting its name and associated members.
```

(Because there will be more)